proxy/http/remap/RemapConfig.cc

```diff
@@ -944,6 +944,9 @@ remap_parse_config_bti(const char *path, BUILD_TABLE_INFO *bti)
 
   std::error_code ec;
   std::string content{ts::file::load(ts::file::path{path}, ec)};
+  if (ec.value() == ENOENT) { // a missing file is ok - treat as empty, no rules.
```

Comment:
Maybe I haven't had enough coffee today, but how is a missing file ok? I know we'd want that to be a failure for our systems.

Reply:
As a general policy for ATS we want to not require the existence of configuration files. If an empty file is acceptable, it shouldn't be required to actually put one there.

Reply:
We have a specific setting to not force having a remap.config -- proxy.config.url_remap.remap_required https://docs.trafficserver.apache.org/admin-guide/files/records.config.en.html#proxy-config-url-remap-remap-required, which has existed since the first commit. I'd expect a missing file to need that setting set to 0.

Reply:
No, that setting means a remap rule is not required to forward the transaction. It has nothing to do with whether there is an actual "remap.config" file.

Reply:
> We have a specific setting to not force having a remap.config -- proxy.config.url_remap.remap_required https://docs.trafficserver.apache.org/admin-guide/files/records.config.en.html#proxy-config-url-remap-remap-required, which has existed since the first commit. I'd expect a missing file to need that setting set to 0.

This also seems correct to me.  Do we at least want a warning in diags.log if remap.config is empty?  Do we know of ATS users who don't do any remapping?